src/rewriter/rewrite_db.cpp

```diff
@@ -36,8 +53,140 @@ void RewriteDb::addRule(DslPfRule id,
                         Node context,
                         bool isFlatForm)
 {
+  NodeManager* nm = NodeManager::currentNM();
+  // flatten if necessary, and if possible
```

Comment:
this seems out of place

Reply:
This is a misplaced comment for a feature that was removed, I'll remove.